diwako_cbrn.vr/scripts/cbrn/functions/fn_threatPFH.sqf

```diff
@@ -79,6 +79,8 @@ if (_player getVariable ["cbrn_using_threat_meter", false]) then {
 };
 
 private _hasChemDetector = "ChemicalDetector_01_watch_F" in (assignedItems _player);
+private _hasGeigerCounter = cbrn_threatGeiger in (items _player); 
```

Comment:
would recommend using the ACE version of hasItem. so you do not need to query the items of your player unit every second. instead it will grab from a pre cached array of pre filtered items instead.

https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_hasItem.sqf